spot_driver/src/spot_driver/spot_ros.py

```diff
@@ -547,36 +551,81 @@ def bodyPoseCallback(self, data):
         mobility_params.body_control.CopyFrom(body_control)
         self.spot_wrapper.set_mobility_params(mobility_params)
 
-    def handle_list_graph(self, upload_path):
+    def handle_start_recording(self, req):
```

Comment:
what is the actual usecase of start/stop recording without using Tablet?

Reply:
It enables to edit and extend the graph used in spot core. If we use Tablet, we have to copy the graph to tablet, edit and extend the graph with tablet and then copy it again to the SpotCore to use it.

Reply:
That is very inefficient.